go/fn/object.go

```diff
@@ -870,7 +870,7 @@ func IsGVK(group, version, kind string) func(*KubeObject) bool {
 
 // GetRootKptfile returns the root Kptfile. Nested kpt packages can have multiple Kptfile files of the same GVKNN.
 func (o KubeObjects) GetRootKptfile() *KubeObject {
-	kptfiles := o.Where(IsGVK(v1.KptFileGroup, v1.KptFileVersion, v1.KptFileKind))
```

Comment:
Ah, I think the kptfile v1 is only merged into kpt repo, while this lib uses https://github.com/GoogleContainerTools/kpt-functions-sdk/blob/master/go/api/kptfile/v1/types.go